profilus project/Program.cs

```diff
@@ -9,16 +9,52 @@
 
 
 
-
-// 
-
 namespace Program
 {
     class Program
     {
+        struct entry  // struct - ключевое слово создание структуры | entriy - Имя структуры.
+        {
+            public entry(string k, int v)
+            {
+                key = k;
+                value = v;
+            }
+            public string key { get; set; } // переменная key, string типа.
+            public int value { get; set; }// переменная value, int типа.
+        };
+        public
+        static string PriemData = "AB =001, BC =002,CD =003, DE =004";
```

Comment:
Can you read this? Do some reply and close thread.

Reply:
Yes, I can. This is easy.